go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/beyondstorage/go-integration-test/v4
 go 1.15
 
 require (
-	github.com/beyondstorage/go-storage/v4 v4.3.2
+	github.com/beyondstorage/go-storage/v4 v4.3.3-0.20210720033749-df6a96af6045
```

Comment:
ping @JinnyYi, Let solve https://github.com/beyondstorage/go-storage/issues/682 so that we can use it here.

Reply:
`go-storage` v4.4.0 have been released.